exonum/src/api/node/mod.rs

```diff
@@ -45,7 +45,7 @@ struct ApiNodeState {
     node_role: NodeRole,
     majority_count: usize,
     validators: Vec<ValidatorKeys>,
-    broadcast_server_address: Option<Addr<websocket::Server>>,
+    broadcast_server_addresses: Vec<Option<Addr<websocket::Server>>>,
```

Comment:
Hmm, It seems that `Option` became redundant in this context.

Reply:
Sure, Thanks for your reviewing. I'll fix it on Tuesday or Wednesday.